system_tests/deployment_test.go

```diff
@@ -62,11 +63,15 @@ func testContractDeployment(t *testing.T, ctx context.Context, client *ethclient
 	nonce, err := client.PendingNonceAt(ctx, accountInfo.Address)
 	Require(t, err)
 
+	// Raise gas fee cap by 25% because new multi-constraint pricing is slightly more aggressive
+	gasFeeCap := new(big.Int).Mul(latestHeader.BaseFee, big.NewInt(5))
```

Comment:
There is a price difference between legacy and new model for `TestExtendedContractDeployment`: 100000000 vs. 125000000. 
@gligneul @eljobe